.github/CODEOWNERS

```diff
@@ -46,15 +46,15 @@ examples/     @StrikerRUS @jameslamb @guolinke
 
 # docker setup
 docker/    @StrikerRUS @jameslamb
-docker/dockerfile-cli    @guolinke @chivee
+docker/dockerfile-cli    @guolinke @chivee @shiyu1994
 docker/gpu/    @huanzhang12
-docker/dockerfile-python    @StrikerRUS @chivee @wxchan @henry0312
+docker/dockerfile-python    @StrikerRUS @chivee @wxchan @henry0312 @shiyu1994
 docker/dockerfile-r    @Laurae2 @jameslamb
 
 # GPU code
 docs/GPU-*.rst    @huanzhang12
-src/treelearner/gpu_tree_learner.cpp    @huanzhang12 @guolinke @chivee
-src/treelearner/tree_learner.cpp    @huanzhang12 @guolinke @chivee
+src/treelearner/gpu_tree_learner.cpp    @huanzhang12 @guolinke @chivee @shiyu1994
+src/treelearner/tree_learner.cpp    @huanzhang12 @guolinke @chivee @shiyu1994
 
 # JAVA code
-swig/    @imatiach-msft
+swig/    @guolinke @chivee @shiyu1994
```

Comment:
Seems that adding users in this file without write access to the repo doesn't work. Proves: #3895, #3538, ...

Reply:
ah makes sense, thanks for noting it